### ISSUE TITLE
[BITV] 9.3.3.2/3.2 - Modal dialog "Choose target folder" - The input field "New folder" lacks visual label and adjacent elements do not explain the function. (2)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#37086]
see also https://github.com/nextcloud/server/issues/37091

- [ ] - [ ] Fix
- [ ] Backport

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-9f92015c98-v3-n2

</details>

- [ ] 9.3.3.2/10.2 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-9f92015c98-v10-n2

**Comment by Pytal** [comment]
Move or copy action not available on master with F2V

Pending implementation

**Comment by AndyScherzinger** [comment]
@Pytal please discuss with @susnux - the modal refered to is the file picker dialog that just got a complete revamp and rewrite from scratch.

So you have multiple ways to trigger that dialog, i.e. Sharing from Nextcloud within a Talk room.

**Comment by Pytal** [comment]
Yes, stable27 looks good
![image](https://github.com/nextcloud/server/assets/24800714/fffd80b2-1992-4584-9a1c-36f59a140511)
but a11y target is master no?

Don't we still need the f2v Move or copy action and the new file picker on master to close @AndyScherzinger?

**Comment by susnux** [comment]
Filepicker should get ready on master next few days, if not today, (need to fix some tests).

**Comment by susnux** [comment]
Done on master with #39792, so should be fixed now
